dags/longitudinal.py

```diff
@@ -40,11 +40,11 @@
                       job_name="Firefox Hardware Report",
                       execution_timeout=timedelta(hours=10),
```

Comment:
The job takes about 3-6 hours to run, so I think 10 hour timeout is appropriate.